assign_rights/serializers.py

```diff
@@ -28,26 +28,58 @@ class RightsShellSerializer(serializers.ModelSerializer):
     """
     start_date = serializers.CharField()
     end_date = serializers.CharField()
-    status = serializers.CharField(source="copyright_status")
-    terms = serializers.CharField(source="license_terms")
-    citation = serializers.CharField(source="statute_citation")
     rights_granted = serializers.ListField(default=[])
-    jurisdiction = serializers.SerializerMethodField()
 
     class Meta:
         model = RightsShell
         fields = (
             "rights_basis",
-            "determination_date",
-            "jurisdiction",
             "start_date",
             "end_date",
             "note",
-            "status",
-            "terms",
-            "citation",
             "rights_granted"
         )
 
+
+class OtherSerializer(RightsShellSerializer):
+    rights_basis = serializers.SerializerMethodField()
```

Comment:
I don't actually know if it's possible, but can you just set this to a value?
```suggestion
    rights_basis = "other"
```

Reply:
Unfortunately, no, it cannot be set directly to a value.

Reply:
Can you do `rights_basis = serializers.CharField(default="other")` ?

Reply:
That doesn't work either.

Reply:
Ah okay, looking at the DRF docs a little more it does seem like a `SerializerMethodField` is the recommended approach.